naett.c

```diff
@@ -34,6 +34,12 @@
 
 #define naettAlloc(TYPE, VAR) TYPE* VAR = (TYPE*)calloc(1, sizeof(TYPE))
 
+#ifdef _MSC_VER 
```

Comment:
Can you move this into the `#ifdef _WIN32` section above?

Reply:
That would be slightly less correct. The fixes are for the microsoft compiler/its headers. They are not windows specific. Although makes not much difference as of now...

Reply:
Fair enough :)